### PR TITLE
Added AdView in HomeFragment.

### PR DESCRIPTION
**Pull request by mehboobali98**
Implemented Banner AdView in the application using AdMob. Test ads are being used as the app has not been published.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -61,6 +61,7 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     implementation 'com.google.firebase:firebase-analytics:17.2.2'
     implementation 'com.google.firebase:firebase-auth:19.3.1'
+    implementation ('com.google.android.gms:play-services-ads:16.+') { force = true; }
 
     implementation 'com.firebaseui:firebase-ui-auth:4.3.1'
     implementation 'androidx.cardview:cardview:1.0.0'
```
Comment (hamadhassan3) [inline app/build.gradle:67]: Nice work, Perfect imports

File: app/src/main/java/com/example/islahmanzilagain/ui/home/HomeFragment.java [hunk 2]
```diff
@@ -43,12 +47,20 @@
...
         homeViewModel =
                 ViewModelProviders.of(this).get(HomeViewModel.class);
         final View root = inflater.inflate(R.layout.fragment_home, container, false);
 
+        Log.d("MyAd", "Before initializing ad.");
+        MobileAds.initialize(getContext(), "ca-app-pub-3940256099942544~3347511713");
+        mAdView = (AdView) root.findViewById(R.id.adView);
+        AdRequest adRequest = new AdRequest.Builder().build();
+        mAdView.loadAd(adRequest);
+
```
Comment (hamadhassan3) [inline app/src/main/java/com/example/islahmanzilagain/ui/home/HomeFragment.java:63]: Try adding more comments please

File: app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -67,15 +67,25 @@
...
+                    android:paddingLeft="40dp"
+                    android:layout_gravity="center_vertical"
+                    app:adUnitId="ca-app-pub-3940256099942544/6300978111"></com.google.android.gms.ads.AdView>
+
+<!--                <TextView-->
+<!--                    android:id="@+id/textView8"-->
+<!--                    android:layout_width="wrap_content"-->
+<!--                    android:layout_height="wrap_content"-->
+<!--                    android:layout_marginTop="20dp"-->
+<!--                    android:layout_marginBottom="20dp"-->
```
Comment (hamadhassan3) [inline app/src/main/res/layout/fragment_home.xml]: Unnecessary comment here

**Review by hamadhassan3 — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (mehboobali98) [reply app/build.gradle:67]: Thanks, boss.

File: app/src/main/res/layout/fragment_home.xml (hunk 3 above)
Comment (mehboobali98) [reply app/src/main/res/layout/fragment_home.xml]: Removed.

File: app/src/main/java/com/example/islahmanzilagain/ui/home/HomeFragment.java (hunk 2 above)
Comment (mehboobali98) [reply app/src/main/java/com/example/islahmanzilagain/ui/home/HomeFragment.java:63]: Done.

**Review by hamadhassan3 — APPROVED** [review]
Well done Sir!

**Pull request merged**